domain/src/main/java/me/androidbox/domain/authentication/remote/Authentication.kt

```diff
@@ -0,0 +1,5 @@
+package me.androidbox.domain.authentication.remote
+
+interface Authentication {
+    suspend fun registerUser(fullName: String, email: String, password: String)
+}
```

Comment:
What's the use of this interface?

Reply:
@MKiperszmid The implementation is in the data layer. Is called AuthenticationImp.